src/Message.php

```diff
@@ -4,14 +4,27 @@
 
 class Message
 {
-    /** @var string */
+    /** @var string|array */
     public $message;
 
     /** @var string */
     public $class;
 
-    public function __construct(string $message, $class)
+    public function __construct($message, $class)
     {
+        if (is_array($message)) {
+            $concat = collect($message)->map(function ($msg) {
+                $child_markup = config('flash-message.child_markup');
+
+                return '<'.$child_markup.'>'.$msg.'</'.$child_markup.'>';
+            })
+            ->implode('');
+
+            $parent_markup = config('flash-message.parent_markup');
```

Comment:
Instead of adding options to specify markup in the config file, let's just a new blade view to our package. Inside that view the rendering can be done. This way, if people want another way to render the flashmessages, they can publish the view an make modifications there.

Reply:
OK that's better